### PR TITLE
ODC 5914 update addflow gherkin scripts

### PR DESCRIPTION
**Pull request by makambalaji**
**Bug Jira Id:**
https://issues.redhat.com/browse/ODC-5914

Description:
updated test scenario [A-08-TC01]
[A-09-TC05 ]- remove or update the step- user will see the list of Providers - Need to Confirm 
Background needs to be re-structured to execute the automation for developer-catalog-details.feature
add-page.feature related scripts are not automated, so updated them with @to-do tag

**Pre-conditions for Epic validation:**

**Checks required for approving Epic gherkin scripts PR:**

- [ ] Add @epic-number to the scenarios or feature file [e.g: @odc-xxx]
- [ ] Add @to-do for automation possible scenarios
- [ ] Add @regression or @smoke based on the importance of the scenario
- [ ] Update the test scenarios count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)
- [ ] Check for the linter issues by executing `yarn run gherkin-lint` on frontend folder [Skip epic number tags related linter issues]

**Refactoring Gherkin scripts criteria for approving PR:**

- [x] Check the linter issues by executing `yarn run gherkin-lint` on frontend folder [Skip epic number tags related linter issues]
- [x] Update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)
- [ ] Add @un-verified tag which are not verified by ODC QE

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature [hunk 1]
```diff
@@ -11,14 +11,15 @@ Feature: Create workload from Operator Backed file
 # Below scenario needs to be executed only once. Second time it throws error. So not recommended to execute in automation suite
         @regression @manual
         Scenario: Create the workload from Operator Backed: A-08-TC01
-            Given user has created or selected namespace "knative-eventing"
+            Given user has not created kafka instance in "knative-eventing" namespace
+              And user has created or selected namespace "knative-eventing"
               And user is at OperatorBacked page
              When user selects knative Kafka card
               And user clicks Create button in side bar
               And user enters name as "knative-kafka" in Create knative Serving page
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature:19]:
```suggestion
              And user enters name as "knative-kafka" in Create KnativeKafka page
```

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature [hunk 2]
```diff
@@ -11,14 +11,15 @@ Feature: Create workload from Operator Backed file
...
               And user clicks create button in Create knative Serving page
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature:20]:
```suggestion
              And user clicks create button in Create KnativeKafka page
```

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature [hunk 3]
```diff
@@ -11,14 +11,15 @@ Feature: Create workload from Operator Backed file
...
              Then user will be redirected to Topology page
-              And user is able to see workload "knative-serving-1" in topology page
+              And user is able to see workload "knative-kafka" in topology page
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature:22]: This doesn't work I can't see "knative-kafka" in topology page

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature (hunk 1 above)
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature:19]: updated

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature (hunk 2 above)
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature:20]: updated

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature (hunk 3 above)
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature:22]: updated

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature [hunk 4]
```diff
@@ -29,3 +30,16 @@ Feature: Create workload from Operator Backed file
...
+        Scenario: Create the Knative eventing workload from Operator Backed: A-08-TC03
+            Given user has installed Jaeger operator
+              And user has created or selected namespace "aut-jaeger"
+              And user is at OperatorBacked page
+             When user selects Jaeger card
+              And user clicks Create button in side bar
+              And user enters name as "jaeger-all-in-one-inmemory1" in Create Jaeger page
+              And user clicks create button in Create Jaeger page
+             Then user will be redirected to Topology page
+              And user is able to see "jaeger-all-in-one-inmemory1" in Topology page
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature:45]: @makambalaji Newline is required

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature [hunk 5]
```diff
@@ -29,3 +30,16 @@ Feature: Create workload from Operator Backed file
               And user clicks Create button in side bar
               And user clicks cancel button in Create knative Serving page
              Then user will be redirected to Operator Backed page from knative Serving page
+
+
+        @regression @manual
+        Scenario: Create the Knative eventing workload from Operator Backed: A-08-TC03
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature:36]:
```suggestion
        Scenario: Create the Jaegar workload from Operator Backed: A-08-TC03
```

**Pull request merged**